packages/database/src/migrations/20210713071004-AddFluTrackerNonFirstNationWithIliMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const REPORT_CODE = 'AU_FLUTRACKING_Percentage_Non_First_Nations_ILI';
+
+const REPORT = {
+  id: generateId(),
+  code: REPORT_CODE,
+  config: {
+    fetch: {
+      aggregations: [
+        'FINAL_EACH_DAY',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'sub_district',
+            aggregationEntityType: 'district',
+          },
+        },
+        'MOST_RECENT',
+      ],
+      dataElements: ['FWV_LGA_004b', 'FWV_LGA_003b'],
+    },
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'aggregate',
+        organisationUnit: 'group',
+        '...': 'last',
+      },
+      {
+        transform: 'select',
+        "'organisationUnitCode'": '$row.organisationUnit',
+        "'% of participants with influenza like illness (ILI)'":
+          'divide($row.FWV_LGA_004b, $row.FWV_LGA_003b)',
+        "'Participants'": '$row.FWV_LGA_003b',
+        "'Participants with influenza like illness (ILI)'": '$row.FWV_LGA_004b',
+      },
+    ],
+  },
+};
+
+const MAP_OVERLAY = {
+  id: REPORT_CODE,
+  name: '% of non-First Nation participants with influenza like illness (ILI)',
+  userGroup: 'Public',
+  dataElementCode: '% of participants with influenza like illness (ILI)',
```

Comment:
It looks fine for me to use a customised value key, but previously all reports use `value`. Perhaps we can make it consistent to other reports?